app/src/main/java/org/eyeseetea/malariacare/network/PushClient.java

```diff
@@ -113,8 +111,8 @@
     //When PushClient is sending the event, the activity is null, becouse PushClient is called in a InstanceService without activity.
     //I can´t access to the activity or context for get the r.string values.
     private static String TAG_IMEI="RuNZUhiAmlv";
```

Comment:
@idelcano it is a minor thing but maybe we can use a better name than TAG_IMEI as now it is going to be the uid for three fields. So that in the future we know what we are storing here.